distribution/docker/druid.sh

```diff
@@ -148,7 +148,7 @@ env | grep ^druid_ | while read evar;
 do
     # Can't use IFS='=' to parse since var might have = in it (e.g. password)
     val=$(echo "$evar" | sed -e 's?[^=]*=??')
-    var=$(echo "$evar" | sed -e 's?^\([^=]*\)=.*?\1?g' -e 's?_?.?g')
+    var=$(echo "$evar" | sed -e 's?^\([^=]*\)=.*?\1?g' -e 's?__?$UNDERSCORE?g' -e 's?_?.?g' -e 's?$UNDERSCORE?_?g')
```

Comment:
Is $UNDERSCORE supposed to be defined somewhere?

Reply:
its a placeholder to handle the escape char